### PR TITLE
Bump PrettyTables to 0.10

### PR DESCRIPTION
**Pull request by tmigot**
This update is necessary to update [`DataFrames.jl`](https://github.com/JuliaData/DataFrames.jl/blob/master/Project.toml) to version 0.22.

### CODE REVIEW

File: src/markdown_tables.jl [hunk 1]
```diff
@@ -40,7 +40,7 @@ Keyword arguments:
 """
 function markdown_table(io :: IO, df :: DataFrame; hl=passfail_highlighter(df), kwargs...)
   header, table = format_table(df, MDformat; kwargs...)
-  pretty_table(io, table, header, tf=markdown, highlighters=hl)
+  pretty_table(io, table, header, tf=tf_markdown, highlighters=hl)
```
Comment (dpo) [inline src/markdown_tables.jl:43]: What is tf_markdown?

File: src/latex_formats.jl [hunk 2]
```diff
@@ -161,7 +161,17 @@ function pretty_latex_stats(io::IO, df::DataFrame;
...
   # that won't work if we put any math headers
-  tf = LatexTableFormat(latex_simple; header_envs=[])
+  tf = LatexTableFormat(
+    top_line       = "\\hline",
+    header_line    = "\\hline",
+    mid_line       = "\\hline",
+    bottom_line    = "\\hline",
+    left_vline     = "|",
+    mid_vline      = "|",
+    right_vline    = "|",
```
Comment (dpo) [inline src/latex_formats.jl:171]: Visually, I find tables look much better without vertical lines. Will this force vertical lines?

File: src/markdown_tables.jl (hunk 1 above)
Comment (tmigot) [reply src/markdown_tables.jl:43]: In the new version of PrettyTables they rename the constant `markdown` as `tf_markdown`

File: src/latex_formats.jl (hunk 2 above)
Comment (tmigot) [reply src/latex_formats.jl:171]: I just kept the default parameters as before, only the constructor has changed here.
On the two examples from the unit tests there are no vertical lines.

Comment (dpo) [reply src/latex_formats.jl:171]: Ok thank you for checking!

**Comment by dpo** [comment]
Looks like `save_stats()` isn't working on Julia Nightly Linux and macOS. I wonder if it's a problem with `tempname()`?!

**Comment by tmigot** [comment]
> Looks like `save_stats()` isn't working on Julia Nightly Linux and macOS. I wonder if it's a problem with `tempname()`?!

I think this is due to JLD2.jl not passing the tests on the nightly Linux and macOS with the same error, [see for instance this workflow](https://github.com/JuliaIO/JLD2.jl/runs/2739585892).
Sorry, I don't know what's the `tempname()`.

**Review by dpo — APPROVED** [review]

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**